tools/validate_rpcs.py

```diff
@@ -238,7 +238,7 @@ def add_tasks_for_height(height: int) -> None:
         for i in range(start_height, end_height + 1):
             add_tasks_for_height(height=i)
             # Make Status Updates.
-            if len(pipeline) >= pipeline_depth:
+            while len(pipeline) >= pipeline_depth:
```

Comment:
when running this tool for all 3 RPC calls, each iteration adds 3 pending tasks, but only one was popped off. Without this change, `pipeline` is allowed to grow indefinitely.